engine/tester/config_parse.c

```diff
@@ -2713,14 +2733,19 @@ get_test_package(xmlNodePtr root, tester_cfg *cfg,
 
     node = root->children;
 
-    /* Get mandatory description */
-    if (node != NULL &&
-        (rc = get_node_with_text_content(&node, "description",
-                                         &pkg->objective)) != 0)
+    if (node != NULL)
     {
-        ERROR("Failed to get mandatory description of the test "
-              "package '%s': %r", pkg->name, rc);
-        return rc;
+        if (pkg->objective != NULL)
+        {
+            node = xmlNodeNext(node);
```

Comment:
This way we simply ignore the first child whatever it be. IMHO it would be better to ensure that it is the description if not  XML_DOC_ASSUME_VALID.